### DISCUSSION TITLE
Philosophy of Park UI

### DISCUSSION THREAD

**Discussion by BlueskyFR** [cschroeter/park-ui#476] [Q&A]
Hey! I am super new to Panda CSS/Ark UI/Park UI.

I am not sure I understand the way this project works: why do we have to copy the components code to our project instead of using them as-is?
For instance I just want to use the Park UI components inside my Nuxt project so why do I have to download (effectively copy-pasting code in my project) instead of just importing a Nuxt module for that?

For example, take [Nuxt UI](https://ui.nuxt.com/getting-started/installation), where only 2 lines are required to use the components.

I am a bit lost, so thanks in advance for your answer!

**Comment by riceboyler** [comment]
I think the best way to express it is that this is akin to shadcn/ui for Tailwind. While they can be used as built with no customization, my take on Park-UI (I'm not involved with the project other than as a very interested user) is that it's a great wrapper on top of Ark-UI that is intended to be a baseline that developers can customize to their own design systems. As such, it doesn't make sense to make one very opinionated import.

**Reply by BlueskyFR** [reply]
So there is no way to import components without effectively "installing" them I guess?
Also it seems that for VueJS there is like only 3-4 components available for installation sooo it makes it a bit weird
